### PR TITLE
revamped signup

### LINKED ISSUE #9: Improve sign up page UI
Enhance the UI/UX of the sign-up page, ensuring it is responsive across all devices and making it visually appealing and interactive.

### PR DESCRIPTION
**Pull request by ShivanshPlays**
closes #9 

made the signup responsive for mobile
Screenshot 2024-10-02 184149

made the signup active in navbar when on signup page
Screenshot 2024-10-02 184210

### CODE REVIEW

**Comment by ChandelAnish** [comment]
@ShivanshPlays  add show password feature

File: client/signup-page/index.html [hunk 1]
```diff
@@ -4,111 +4,199 @@
...
+                max-width: 500px;
+            }
+        }
+
+        @media (min-width: 769px) {
+            .container {
+                max-width: 600px;
+            }
+        }
+    </style>
```
Comment (ChandelAnish) [inline client/signup-page/index.html:103]: write the css in the style.css file

**Comment by ShivanshPlays** [comment]
Hi, please check I have done the required changes

**Comment by ShivanshPlays** [comment]
@sanjay-kv @ChandelAnish Please review and merge.

**Review by ChandelAnish — COMMENTED** [review]
->show the error message in red color
->align the signup card in center and remove the vertical scroll
-> improve dropdown UI ,use bootstrap or tailwind

image

**Comment by ShivanshPlays** [comment]
Hi, @ChandelAnish 
I have done the necessary changes, please check.

Screenshot 2024-10-06 162420

**Review by ChandelAnish — COMMENTED** [review]
You have made some serious issues, even though the password is wrong it still goes to the OTP page and many more issues. You were assigned only to make UI-related change but you have hampered the core functionality.
Only do the UI-related changes

https://github.com/user-attachments/assets/ba7d01e1-3bca-4d37-94d3-37ca6387a329

**Comment by ShivanshPlays** [comment]
Apologies, I'll do the necessary changes

**Pull request closed without merging**